Updates to cacti config and cacti patch to address issues with OS/GCC differences.

stdavids opened the pull request.

taylor-bsg commented:
For posterity, what was the actual incompatibility? A little hard to find
in the change log..

On Tue, Apr 7, 2020 at 3:53 PM Scott Davidson <notifications@github.com>
wrote:

> Merged #7 <https://github.com/bespoke-silicon-group/bsg_fakeram/pull/7>
> into master.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/bespoke-silicon-group/bsg_fakeram/pull/7#event-3211899161>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AEFG5ACPR6LU3XHZZCIY7CTRLOVGZANCNFSM4MDPO6NA>
> .
>

stdavids commented:
The aspect ratio from cacti, which was a value I added to the report, seemed to inconsistent OS and versions of GCC. Presumably it was deallocated and was being corrupted. I replaced this with height and width attributes which seem to still be allocated on the few OS/gcc versions I tried.
